lib/src/dart_fix.dart

```diff
@@ -193,6 +204,74 @@ Future<ExitResult> dartFixInternal(
   return ExitResult.ok;
 }
 
+Future<ExitResult> checkPackage(
+    Logger logger, String packageName, bool performDryRun) async {
+  logger.stdout('Checking package $packageName...');
+
+  String jsonData =
+      await _uriDownload('https://pub.dartlang.org/api/packages/$packageName/');
+  Map json = jsonDecode(jsonData);
+  Map pubspec = json['latest']['pubspec'];
+  String homepage = pubspec['homepage'];
```

Comment:
Put this in a URI? Will make later code easier – less splitting, startsWith, tec...